docs/TRY_IT_OUT_EXECUTOR.md

```diff
@@ -0,0 +1,219 @@
+#### TryItOut Executor
+
+TryItOut Executor is an OpenAPI specific HTTP client for OAS operations.
+It maps an OAS operation and values into an HTTP request.
+
+To execute an OAS operation, you need to provide TryItOut Executor with various properties.
+
+Type notations are formatted like so:
+
+- `String=""` means a String type with a default value of `""`.
+- `String=["a"*, "b", "c", "d"]` means a String type that can be `a, b, c, or d`, with the `*` indicating that `a` is the default value.
+
+Property | Description
+--- | ---
+`spec` | `Object`, **REQUIRED**. OpenAPI definition represented as [POJO](https://en.wikipedia.org/wiki/Plain_old_Java_object).
+`operationId` | `String`. Unique string used to identify an operation. If not provided, `pathName` + `method` must be used instead.
+`pathName` | `String`. OpenAPI defines a unique operation as a combination of a path and an HTTP method. If `operationId` is not provided, this property must be set.
+`method` | `String=["GET", "HEAD", "POST", "PUT", "DELETE", "CONNECT", "OPTIONS", "TRACE", "PATCH"]`. OpenAPI defines a unique operation as a combination of a path and an HTTP method. If `operationId` is not provided, this property must be set.
+`parameters` | `Object`. Parameters object, eg: `{ q: 'search string' }`. Not recognized parameters (not defined in `spec`) will be ignored.
+`securities` | `Object`. Maps security schemes to a request. Not recognized securities will ge ignored. <br/><br/>*Examples*<br /><br /> *Bearer:* `{ authorized: { BearerAuth: {value: "3492342948239482398"} } }` <br /><br /> *Basic:* `{ authorized: { BasicAuth: { username: 'login', password: 'secret' } } }` <br /><br /> *ApiKey:* `{ authorized: { ApiKey: { value: '234934239' } } }` <br /><br /> *oAuth2:* `{ authorized: { oAuth2: { token: { access_token: '234934239' } } } }`
+`requestContentType` | `String`. Sets [appropriate media type](https://swagger.io/docs/specification/describing-request-body/) for request body, e.g. `application/json`. If supplied media type is not defined for the request body, this property is ignored.
+`responseContentType` | `String`. Expect [appropriate media type](https://swagger.io/docs/specification/describing-responses/) response, e.g. `application/json`. Creates an `Accept` header in `Request` object.
+`http` | `Function=Http`. A function with an interface compatible with [HTTP Client](HTTP_CLIENT.md).
+`userFetch` | `Function=cross-fetch`. Custom **asynchronous** fetch function that accepts two arguments: the `url` and the `Request` object and must return a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) object. More info in [HTTP Client](HTTP_CLIENT.md) documentation.
+
+For all later references, we will always use following OpenAPI 3.0.0 definition when referring
+to a `spec`.
+
+```js
```

Comment:
I think our preference is to encourage YAML for api definitions, rather than json.

Reply:
Yesp, `yaml` is more compact and readable. Converted to yaml. THanks